libs/ff-three/source/HTMLSpriteGroup.ts

```diff
@@ -69,18 +77,19 @@ export default class HTMLSpriteGroup extends Object3D
      * @param container HTML container element for the HTML elements.
      * @param camera The camera used to render the 3D scene.
      */
-    render(container: HTMLElement, camera: Camera)
+    render(viewport: IViewport, camera: Camera)
     {
         if (!this.visible) {
             return;
         }
-
+        //Only get bounds once to prevent forced reflows while looping
+        const bounds = new DOMRect(viewport.x, viewport.y, viewport.width, viewport.height);
```

Comment:
Still need to test a little more, but looks like this line should be `const bounds = viewport.overlay.getBoundingClientRect();`

Reply:
Yes it should. I thought they were the same, and they are in voyager-explorer, but not in a story page. 

There seems to be no significant performance hit when introducing this call so everything should be all right.